src/components/Form/Form.js

```diff
@@ -40,48 +40,52 @@ function Form ({title, artist, coverImg, id, showModal}) {
   }
 
   return (
-    <form name='journal-form' className='journal-form'>
-      <div className='form-container'>
-        <img className='form-image' src={coverImg} alt={`${title} by ${artist}`} onError={onImageError}/>
-        <div className='form-info'>
-          <p><span className='form-title'>{title}</span>  {artist}</p>
-          <p className='form-listen'>Listened</p>
-          <div className='date-container'>
-            <Flatpickr 
-              name='date-input'
-              className='date-input'
-              value={date}
-              options={
-                { altFormat: true,
-                  dateFormat: "F j, Y",
-                  maxDate: "today",
-                  disableMobile: true,
-                  onChange: function updateDate (selectedDates, dateStr) {
-                    setDate(dateStr)
-                  }
-                }}
-              placeholder={formattedDate}/>
-            <img className='calendar-icon'  src={process.env.PUBLIC_URL + "/images/calendar.png"} alt='calendar-icon'/>
+    <div className='main-form-container'>
+      <form name='journal-form' className='journal-form'>
+        <div className='form-container'>
+          <img className='form-image' src={coverImg} alt={`${title} by ${artist}`} onError={onImageError}/>
+          <div className='form-info'>
+            <p><span className='form-title'>{title}</span></p>
+            <p>{artist}</p>
+            <p className='form-listen'>Listened</p>
+            <div className='date-container'>
+              <Flatpickr 
+                name='date-input'
+                className='date-input'
+                value={date}
+                options={
+                  { altFormat: true,
+                    dateFormat: "F j, Y",
+                    maxDate: "today",
+                    disableMobile: true,
+                    onChange: function updateDate (selectedDates, dateStr) {
+                      setDate(dateStr)
+                    }
+                  }}
+                placeholder={formattedDate}/>
+                <img className='calendar-icon'  src={process.env.PUBLIC_URL + "/images/calendar.png"} alt='calendar-icon'
+              />
+            </div>
           </div>
+          <img className='form-escape' src={process.env.PUBLIC_URL + "/images/escape-white.png"} alt='escape-icon' onClick={showModal} tabIndex={1}/>
         </div>
-        <img className='form-escape' src={process.env.PUBLIC_URL + "/images/escape-white.png"} alt='escape-icon' onClick={showModal}/>
```

Comment:
what is form-escape do as functionality within the modal?

Reply:
It's there to style the exit button